eng/scripts/Language-Settings.ps1

```diff
@@ -245,3 +245,178 @@ function GetExistingPackageVersions ($PackageName, $GroupId=$null)
     return $null
   }
 }
+
+function Get-dotnet-DocsMsMetadataForPackage($PackageInfo) {
+  $suffix = ''
+  $parsedVersion = [AzureEngSemanticVersion]::ParseVersionString($PackageInfo.Version)
+  if ($parsedVersion.IsPrerelease) { 
+    $suffix = '-pre'
+  }
+
+  New-Object PSObject -Property @{ 
+    DocsMsReadMeName = $PackageInfo.Name.ToLower() -replace "." , ""
+    LatestReadMeLocation = 'api/overview/azure'
+    PreviewReadMeLocation = 'api/overview/azure'
+    Suffix = $suffix
+  }
+}
+
+
+# Details on CSV schema:
+# https://review.docs.microsoft.com/en-us/help/onboard/admin/reference/dotnet/documenting-nuget?branch=master#set-up-the-ci-job
+function Get-DocsCiConfig($configPath) {
+  Write-Host "Loading csv from $configPath"
+  $output = @()
+  foreach ($row in Get-Content $configPath) { 
```

Comment:
Might be worth a comment about some of the magic parts to at least call out the expectation of what we are looking for. Also worth adding a note of why we are parsing the CSV file ourselves instead of using the built-in parser.